_api-reference/list/list-shards.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: default
+title: List Shards
+parent: List API
+nav_order: 20
+---
+
+# List Shards
+**Introduced 2.18**
+{: .label .label-purple }
+
+The List Shards operation outputs, in a paginated format, the state of all primary and replica shards and how they are distributed.
+
+## Path and HTTP methods
+
+```json
+GET _list/shards
+GET _list/shards/<index>
+```
+
+## Query parameters
+
+All parameters are optional.
+
+Parameter | Type | Description
+:--- | :--- | :---
+`bytes` | Byte size | Specifies the units for the byte size, for example, `7kb` or `6gb`. For more information, see [Supported units]({{site.url}}{{site.baseurl}}/opensearch/units/).
+`local` | Boolean | Whether to return information from the local node only, instead of from the cluster manager node. Default is `false`.
+`cluster_manager_timeout` | Time | The amount of time to wait for a connection to the cluster manager node. Default is 30 seconds.
+`cancel_after_time_interval` | Time | The amount of time after which the shard request is canceled. Default is `-1` (no timeout).
+`time` | Time | Specifies the units for time, for example, `5d` or `7h`. For more information, see [Supported units]({{site.url}}{{site.baseurl}}/opensearch/units/).
+`next_token` | String | Fetches the next page of indexes. When `null` provides the first page of indexes. Default is `null`. 
```

Comment:
"Fetches the next page of indexes when `null` provides the first page of indexes"?